controllers/events.js

```diff
@@ -259,6 +260,35 @@ const kickoutPeer = async (req, res) => {
   }
 };
 
+const generateEventCode = async (req, res) => {
```

Comment:
The `code` here we're taking from req.body, the `eventCodeUuid` is for generating uuid for the event code object.
So that it can be used as identifier.
If we look into the controller file I had take eventCode and role from body.